### PR TITLE
[WAF] Change "Block" to "Drop" in waf-fields.md?

### PR DESCRIPTION
**Pull request by chrismo**
I don't see "drop" documented here, but I do see it in the actual logs, instead of "block" I'm presuming. But I'm not really sure what the docs _should_ be.

### CODE REVIEW

File: content/logs/reference/waf-fields.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The Web Application Firewall (WAF) contains rules managed by Cloudflare to block
 | ------------------------------------------ | --------------- | -------------------------------------------- |
```
Comment (sh471) [inline content/logs/reference/waf-fields.md:16]: content/logs/reference/waf-fields.md

File: content/logs/reference/waf-fields.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The Web Application Firewall (WAF) contains rules managed by Cloudflare to block
...
 | <span style="font-weight: 400;">`0`</span> | Unknown         | Take no other action.                        |
 | <span style="font-weight: 400;">`1`</span> | Allow           | Bypass all subsequent WAF rules.             |
-| <span style="font-weight: 400;">`2`</span> | Block           | Block with an HTTP 403 response.             |
```
Comment (sh471) [inline content/logs/reference/waf-fields.md:19]: <span style="font-weight: 400;">`2`</span> | Drop          | Block with an HTTP 403 response.

Comment (sh471) [reply content/logs/reference/waf-fields.md:19]: I hope that was close enough ??

File: content/logs/reference/waf-fields.md (hunk 1 above)
Comment (sh471) [inline content/logs/reference/waf-fields.md:16]: <span style="font-weight: 400;">`2`</span> | Drop            | Block with an HTTP 403 response.

File: content/logs/reference/waf-fields.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The Web Application Firewall (WAF) contains rules managed by Cloudflare to block
...
+| <span style="font-weight: 400;">`2`</span> | Drop            | Block with an HTTP 403 response.             |
```
Comment (sh471) [inline content/logs/reference/waf-fields.md:19]: #7369

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**